spec/requests/stories/articles_search_spec.rb

```diff
@@ -6,5 +6,19 @@
       get "/search?q=hello"
       expect(response.body).to include("=> Search Results")
     end
+
+    context "with non-empty query" do
+      it "renders page with default page title" do
```

Comment:
Aren't the `it` descriptions reversed here? Isn't this one `"renders page with query in page title"` and the other one is `"renders page with default page title"`?

Reply:
You're right, my bad, thanks for catching that! I'm on it.

Reply:
Having some `asdf` and ruby version issues locally that I can't seem to be able to fix 🤦🏻‍♀️ I made these changes via the web editor. Should be updated now.